src/praisonai/praisonai/cli/commands/config.py

```
@@ -384,3 +446,155 @@ def config_doctor():
                 "size": cache_size,
             }
         })
+
+
+@app.command("show")
+def config_show(
+    format: str = typer.Option(
```

Ruff (0.15.17)

[error] Function argument `format` is shadowing a Python builtin

(A002)